### PR TITLE
test code for decorators

### LINKED ISSUE #20: [Feature] Make test code for existing code
## Feature Details
- [ ] Decorator e2e test code

```typescript
  test('The @Delete decorator is routed to the Http get method.', async () => {
    // given
    @RestController()
    class TestController {
      @Delete('/delete')
      deleteMethod() {
        return 'delete';
      }
    }
    // when
    const response = await request(app).delete('/delete');

    // then
    expect(response.statusCode).toEqual(200);
    expect(JSON.parse(response.text)).toBe('delete');
  });
```

### PR DESCRIPTION
**Pull request by lsh23**
## Summary
test code for decorators

- View Controller
  - Controller
- HTTP Method
  - Get
  - Post
  - Put
  - Patch
  - Delete
  - Optoins
  - Head
  - Trace
- HttpCode
- Argument resolver
  - Req
  - Res
  - Cookie
  - Param
  - Query

## Additional context
todo
  - Body
  - Connect
  - UseGuard
  - AuthenticatedUser

## Close
- close #20

### CODE REVIEW

File: src/controller.ts [hunk 1]
```diff
@@ -495,7 +495,7 @@ export function Head(url: string): MethodDecorator {
  * @returns {MethodDecorator} - A MethodDecorator tailored to denote that the adorned method should
  * handle incoming CONNECT requests for the provided URL path.
  */
-export function CONNECT(url: string): MethodDecorator {
+export function Connect(url: string): MethodDecorator {
```
Comment (Dltmd202) [inline src/controller.ts:498]: Why do you only use capital letters for `Connect`?😲

**Review by Dltmd202 — APPROVED** [review]
You worked really hard👍

File: src/controller.ts (hunk 1 above)
Comment (lsh23) [reply src/controller.ts:498]: it was a just typo. 😅

Comment (Dltmd202) [reply src/controller.ts:498]: Ah, I misunderstood that it was changed from uppercase to lowercase.

**Pull request merged**